### PR TITLE
👷‍♀️FIX: file path building

### PR DESCRIPTION
**Pull request by kenjinp**
now when creating a resource instance with `Resource.getSelf` the sdk will check for the `files` path and parse the url correctly. Before this caused problems because file resources don't have schemas in their paths.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbp/nexus-sdk",
-  "version": "1.0.0-beta.21",
+  "version": "1.0.0-beta.22",
```
Comment (julienmachon) [inline package.json]: It's fine just now but don't change the package version in a PR like this. We change it when we want to do a release, which might include several PRs

File: src/Resource/utils.ts [hunk 2]
```diff
@@ -23,6 +23,8 @@ import { SparqlViewQueryResponse } from '../View/SparqlView/types';
 import { getSparqlView } from '../View/utils';
 import Store from '../utils/Store';
 
+const FILES_PATH_LABEL = 'files';
```
Comment (julienmachon) [inline src/Resource/utils.ts]: what is the selfUrl is `http://api.v1/resources/org/label/_/I-love-files-but-i-am-not-one-of-them`?

Comment (kenjinp) [reply src/Resource/utils.ts]: oh no!

Comment (kenjinp) [reply src/Resource/utils.ts]: I added a test to describe this scenario

**Pull request merged**